addons/ai/functions/fnc_mortarIsKindOf.sqf

```diff
@@ -0,0 +1,21 @@
+/*
+    Author:
+        Adam Bridgford
+
+    Description:
+        Checks what kind of unit caller is then adds appropiate delay
+
+    Parameter(s):
+        0: Unit <OBJECT>
+
+    Return Value:
+        Nothing
+*/
+
+if (_unit isKindOf ["CUP_O_RU_JTAC_EMR","CUP_O_RU_JTAC_Flora"]) then {
```

Comment:
```_unit``` does not refer to anything in the scope of this function. If you're passing a unit as a parameter to this function, access it here with ```params```